libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/IpvSessionService.java

```diff
@@ -109,6 +109,20 @@ public IpvSessionItem getIpvSessionByAccessToken(String accessToken)
                 });
     }
 
+    public IpvSessionItem getIpvSessionByCriOAuthSessionId(String criOAuthSessionId)
+            throws IpvSessionNotFoundException {
+        return callRunTaskWithBackoff(
+                () -> {
+                    var item = dataStore.getItemByIndex("criOAuthSessionId", criOAuthSessionId);
```

Comment:
Interesting that we have this index but we don't appear to use it anywhere at the moment

Reply:
As it turns out the IpvSession table _doesn't have a column for criOAuthSessionId?_ 

With this, I will replace the criOAuthSessionId with the clientOAuthSessionId for this, and as a GSI as criOAuthSessionId is unused.